recipes/r-spatest/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = '2.0.2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-spatest
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: SPAtest_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/SPAtest_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/SPAtest/SPAtest_{{ version }}.tar.gz
+
+
+  sha256: 191843f66f626d3279523afddfdc9ad8561184acbd6842cce07d86a6a37c38c0
+  # patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  skip: true  # [win]
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
+    - $R -e "library('SPAtest')"  # [not win]
+    - "\"%R%\" -e \"library('SPAtest')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
+
+about:
+  home: https://CRAN.R-project.org/package=SPAtest
+  license: GPL (>= 2)
```

Comment:
If it is GPLe the license needs to be distributed with the code.

Reply:
Thanks.